web/assets/themes/wp-berlin/footer.php

```diff
@@ -12,10 +12,10 @@
     <div class="footer-inner">
     <ul class="menu">
         <li class="menu-item">
-            <a href="">Impressum</a>
+            <a href="/imprint">Impressum</a>
         </li>
         <li class="menu-item">
-            <a href="">Topics</a>
+            <a href="/topics">Topics</a>
```

Comment:
This file does not exist. For the moment being, the topics are in https://tech.wpmeetup-berlin.de/contribute/#speaking @svenbrier could you please update this PR accordingly?